doc/api/util.md

```diff
@@ -1249,6 +1249,23 @@ util.types.isMapIterator(map.entries());  // Returns true
 util.types.isMapIterator(map[Symbol.iterator]());  // Returns true
 ```
 
+### util.types.isModuleNamespaceObject(value)
+<!-- YAML
+added: v10.0.0
+-->
+
+* Returns: {boolean}
+
+Returns `true` if the value is an instance of a [Module Namespace Object][].
+
+For example:
+
+```js
+import('./a.js').then((ns) => {
+  util.types.isModuleNamespaceObject(ns); // Returns true
```

Comment:
(Not necessarily true if there is a `then` export.)

I’d use an `import * as ns` example.

Reply:
@devsnek Then IMO we should ignore the block.

Reply:
does it look okay at this point?